client/src/client.rs

```diff
@@ -534,6 +498,53 @@ impl Client {
     }
 }
 
+/// Client implements a JSON-RPC client for the Bitcoin Core daemon or compatible APIs.
+pub struct Client {
+    client: jsonrpc::client::Client,
+}
+
+impl Client {
+    /// Creates a client to a bitcoind JSON-RPC server.
+    pub fn new(url: String, user: Option<String>, pass: Option<String>) -> Self {
+        debug_assert!(pass.is_none() || user.is_some());
+
+        Client {
+            client: jsonrpc::client::Client::new(url, user, pass),
+        }
+    }
+
+    /// Create a new Client.
+    pub fn from_jsonrpc(client: jsonrpc::client::Client) -> Client {
+        Client {
+            client: client,
+        }
+    }
+}
+
+impl RpcApi for Client {
+    /// Call an `cmd` rpc with given `args` list
+    fn call<T: for<'a> serde::de::Deserialize<'a>>(
+        &self,
+        cmd: &str,
+        args: &[serde_json::Value],
+    ) -> Result<T> {
+        // Get rid of to_owned after
+        // https://github.com/apoelstra/rust-jsonrpc/pull/19
```

Comment:
Is this the correct pull request because it's already merged?

Reply:
Yeah the PR got merged but it's not included in a release yet. It's a minor thing and we're waiting for some bigger upstream change in the jsonrpc crate that is more important than pushing for a release just for this change. But thanks for reminding me :)